browser/src/control/jsdialog/Util.Dropdown.js

```diff
@@ -41,6 +41,9 @@ JSDialog.OpenDropdown = function (id, popupParent, entries, innerCallback, popup
 		]
 	};
 
+	this.popupParent = popupParent;
+	this.popupParent.setAttribute('aria-expanded', true);
```

Comment:
I believe you can drop `this.popupParent` saving here... also inside `JSDialog.` or `app.`.
Please see in this function you have `generateCallback`, that  callback handles all signals from the dropdown.
You can catch there cases where dropdown should be closed and - setup correct property for `popupParent`.

Reply:
It looks like an old commit

Reply:
Fixed